python/semantic_kernel/memory/volatile_memory_store.py

```diff
@@ -50,3 +50,47 @@ async def get_nearest_matches_async(
         # Then, take the top N results
         top_results = filtered_results[:limit]
         return top_results
+
+    def compute_similarity_scores(
+        self, embedding: ndarray, embedding_array: ndarray
+    ) -> ndarray:
+        """Compute the similarity scores between the
```

Comment:
Nit: we should all discuss the format we want docstrings to be in. 

I've been using Copilot's suggestions to help me generate docstrings, but I've noticed that I haven't been consistent (sometimes it's `:param x: blah` style; other times, it's `Args:\n...` style) 

Let's all discuss and make sure docstrings are consistent at some point in the near future.